File: 12.05/flip.py
def solution(n):
    answer = ''

    while n > 0:			
        n, b = divmod(n,3)	# n을 3으로 나눈 몫과 나머지 a , b에저장
        answer += str(b) # 배열에 저장 
    return int(answer, 3) # 진법변환 

File: lvl1/12.05/flip.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <math.h>//pow 함수 써보기 

int solution(int n) 
{
    int answer = 0;
    int len = 1;
    int div = n;

    while(div/3>0)
    {
        div /= 3;
        len++;  //len 길이구해주기
    }

    for(int i=1; i<=len; i++){
        answer += (n%3) * pow(3,len-i); //pow ( a,b) a제곱 b a^b
        n /= 3;
    }


    return answer;
}
int main()
{

    int  n =45;

    printf("%d", solution(n));
}